src/Symfony/Component/Console/EventListener/ErrorListener.php

```diff
@@ -75,7 +75,8 @@ public static function getSubscribedEvents()
 
     private static function getInputString(ConsoleEvent $event)
     {
-        $commandName = $event->getCommand() ? $event->getCommand()->getName() : null;
+        $command = $event->getCommand();
```

Comment:
Not much gain here I think, but anyway, this could go in 3.4 branch.

Reply:
I would revert this change altogether.

Reply:
Since this is an error listener, we'll not gain really much. Ok, will revert.